### ISSUE TITLE
No Likes for Tags in Mode 1 through 3

### ISSUE THREAD

**Issue by Markus978** [instabot-py/instabot.py#440]
Hi there,

I got to set instabot up and running and was testing the different modes. Everything works fine and the follow/unfollow is working with my set delay. 
However, it appears that only in Mode=0 the bot is giving Likes. All other modes just follow/unfollow. 

How and what would I have to adjust that the bot would also give Likes in the other modes?

Thank you for your help, Markus

**Comment by ChipWolf** [comment]
Mode 1 is a mess, mode 3 only unfollows people.
Mode 2 should be liking.

**Comment by Markus978** [comment]
I agree on Mode 1 🙈 But it seems to work if there is no need for the likes.
As for Mode 2, I thought so, too, and it does follow, unfollow and like.
BUT for whatever reason in Mode 2 the bot stops to unfollow people after a certain amount of time. And it is not related to the set time I set until it is supposed to start unfollowing... very strange. Cant figure out why.

**Comment by ChipWolf** [comment]
Let me rephrase; all of the modes are a mess, a lot of refactoring is needed. I'd stick with 0 personally

**Comment by ugurozturk** [comment]
As he said mod 1 and mod 3 is not liking
